types/prop-types/prop-types-tests.ts

```diff
@@ -0,0 +1,13 @@
+import * as PropTypes from "prop-types";
+
+let testObj: any = {
```

Comment:
This doesn't really test out the library. All you've done is put everything in a bag of type `any`. If the types were wrong, this code would still work. Try including realistic uses of the provided functions and objects.